test cases/diasdk/1 simple/meson.build

```diff
@@ -0,0 +1,8 @@
+project('diatest', 'cpp')
+
+if not ['msvc', 'clang-cl'].contains(meson.get_compiler('cpp').get_id())
```

Comment:
This is different than what is implemented internally, which says "clang" and "msvc"`